scripts/test/e2e/load-busybox

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+src=busybox@sha256:3e8fa85ddfef1af9ca85a5cfb714148956984e02f00bec3f7f49d3925a91e0e7
+dest=registry:5000/busybox:1.27.2
+docker pull $src
```

Comment:
wondering; is this pulling from hub? if so, should we consider having a "frozen image" baked in the base image (like we have in moby/moby https://github.com/moby/moby/blob/2bc9bff86707737ff5df5f23b613d16281f9d223/Dockerfile.e2e#L16-L21)?

Reply:
I was following the same `load-alpine` script we have that pulls by digest.  Up to @dnephin on the best path forward here, but I think that makes sense in a separate PR.

Reply:
Ya, we could do that. I agree we should do it in a separate PR.

One thing I'd like to avoid is that in moby/moby we have about 4 or 5 different ways of loading fixtures. Right now everything is loaded from `setup()` in `scripts/test/e2e/run`. We can move some image freezing to the Dockerfile, but it will couple the run script to the Dockerfile.

Reply:
Yes - I agree ok looking at that in a separate PR; just want to prevent our test from relying on Docker Hub too much :+1: